Packs/AWS-SNS/ReleaseNotes/1_0_7.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### AWS - SNS
+
+- Updated the pack to be XSOAR supported.
```

Comment:
@ShirleyDenkberg wdyt about the text for adopting a community pack?

Reply:
Updated the pack to be Cortex XSOAR supported.